src/test/GradleTestDevModeActions.ts

```diff
@@ -263,6 +266,21 @@ it('View test report for gradle project', async () => {
 }).timeout(30000);
 
 
+  // Now, clear the command history of the "command palette" to avoid receiving "recently used" suggestions. This action should be performed at the end of Gradle Project tests.
+it('Clear Command Palatte', async () => {
+  await new Workbench().executeCommand('Clear Command History');
```

Comment:
can we move this as a function in testUtils class so that we can reuse this whenever we want.

Reply:
Yes, moved to testUtils class